generator/generator.go

```diff
@@ -247,9 +248,14 @@ func (img *Image) appendInitBinary(initBinary string) error {
 
 func (img *Image) appendExtraFiles(binaries []string) error {
 	for _, f := range binaries {
-		if !strings.HasPrefix(f, "/") {
-			// simple names like "strace" are resolved as binaries under /usr/bin
-			f = "/usr/bin/" + f
+		if !filepath.IsAbs(f) {
+			// If the given name is not an absolute path, assume that it refers
+			// to an executable and lookup the path to the executable using $PATH.
+			var err error
+			f, err = exec.LookPath(f)
+			if err != nil {
+				return err
+			}
```

Comment:
I think we need to make sure that the file is still added to /usr/bin afterwards, even if originates it a different location on the host.

Reply:
This is one option indeed.

Another option is to add binaries with `/usr/sbin/` path and then modify `PATH` envvar in ` boost()` function.

Reply:
I have added an `img.AppendFileTo` function and started using this for adding binaries. Haven't tested this extensively yet, but at least it seems to work for `setfont`. Let me know what you think.

<!--
Also: While at it, should we address the hardcoded busybox path in `init` as well or should we somehow ensure that Busybox is always added as `/bin/busybox` to the ramdisk with the `img.AppendFileTo` function?

https://github.com/anatol/booster/blob/edeff4c705d5f2b3a02162bed74626b70754ca63/init/main.go#L790-L796
-->